### ISSUE TITLE
etcd version >= 3.1.12

### ISSUE THREAD

**Issue by ThomasGsp** [kubernetes/website#8098]
**This is a...** 

- [X ] Bug Report

**Problem:**
etcd version proposed is not valid:

```
root:~ #  kubeadm init --config=config.yaml
[init] Using Kubernetes version: v1.10.1
[init] Using Authorization modes: [Node RBAC]
[preflight] Running pre-flight checks.
	[WARNING FileExisting-crictl]: crictl not found in system path
Suggestion: go get github.com/kubernetes-incubator/cri-tools/cmd/crictl
[preflight] Some fatal errors occurred:
	[ERROR ExternalEtcdVersion]: this version of kubeadm only supports external etcd version >= 3.1.12. Current version: 3.1.10
	[ERROR ExternalEtcdVersion]: this version of kubeadm only supports external etcd version >= 3.1.12. Current version: 3.1.10
[preflight] If you know what you are doing, you can make a check non-fatal with `--ignore-preflight-errors=...`
```

**Proposed Solution:**
Change the version for 3.1.12, in section "Run etcd" part "systemd"
```
export ETCD_VERSION=v3.1.12
curl -sSL https://github.com/coreos/etcd/releases/download/${ETCD_VERSION}/etcd-${ETCD_VERSION}-linux-amd64.tar.gz | tar -xzv --strip-components=1 -C /usr/local/bin/
rm -rf etcd-$ETCD_VERSION-linux-amd64*

```
**Page to Update:**
https://kubernetes.io/docs/setup/independent/high-availability/#run-etcd

**Comment by mikebryant** [comment]
Fixed by #8099?

**Comment by tengqm** [comment]
Confirm the issue has been fixed.

/close
